index.js

```diff
@@ -106,7 +106,7 @@ const stylistic = {
   'eol-last': 2,
 
   // Two-space indentation, always
-  indent: [2, 2],
+  indent: [2, 2, {'SwitchCase': 1}],
```

Comment:
Technically the key doesn't need to be quoted here - if were were running this ruleset against itself, it would complain